public/graph.js

```diff
@@ -360,6 +361,24 @@ class GraphController {
             }
             const tickValuesForXAxis = dataFilteredWeekGroupedDaily.map(d => d.datetime);
 
+            // Add the X gridlines
```

Comment:
This makes more sense on 10 minutes view, the 1-day gridlines view makes the graph look too busy and congested, after all we have spacing between the bars so its already clear where each day starts.

Reply:
I will reduce the opacity of x gridlines in one day view